### DISCUSSION TITLE
MongoDB compatibility

### DISCUSSION THREAD

**Discussion by opsingh861** [Azure/data-api-builder#2441] [Q&A] [answered]
In documentation NoSQL is mentioned, so does DAB supports MongoDB which is a NoSQL database.

**Comment by sajeetharan** [comment, marked as answer]
https://github.com/Azure/data-api-builder/discussions/1342

**Reply by opsingh861** [reply]
Thank you @sajeetharan for replying to my query. I have already looked into that, but that is from long back more then one year. So I wanted to know the current situation.

**Reply by sajeetharan** [reply]
You're welcome, can you write to me at sasinnat@microsoft.com with your requirement, would like to discuss over a call! thanks

**Reply by opsingh861** [reply]
Yeah sure, It would be great.

**Reply by opsingh861** [reply]
Hey @sajeetharan, I have sent you my requirements on your email. Maybe it's in your spam, could you please check?
